frontend/src/modules/machinery/components/Translation.js

```diff
@@ -53,7 +55,9 @@ export default function Translation(props: Props) {
             return;
         }
 
-        if (typeof editorContent !== 'string') {
+        if (!entity) {
+            addTextToEditorTranslation(translation.translation);
```

Comment:
Since this is a special case, it would be nice to add a short comment on top explaining why we are doing it.